lib/measured/unit_system_builder.rb

```diff
@@ -8,12 +8,36 @@ def unit(unit_name, aliases: [], value: nil)
     nil
   end
 
+  def si_unit(unit_name, aliases: [], value: nil, range: range = 3)
+    @units.concat build_si_units(unit_name, aliases: aliases, value: value, range: range)
+    nil
+  end
+
   def build
     Measured::UnitSystem.new(@units)
   end
 
   private
 
+  def build_si_units(name, aliases: [], value: nil)
+    si_units = []
+    get_positive_prefixes(range).each_with_index do |prefix, index|
+      si_units << build_si_unit(prefix + name, concat_prefixes(aliases, prefix), value * power(index))
+    end
+
+    get_negative_prefixes(range).each_with_index do |prefix, index|
+      si_units << build_si_unit(prefix + name, concat_prefixes(aliases, prefix), value * power(index, -1))
+    end
```

Comment:
Thanks! This is a great idea since the prefixes that you suggested to add are not evenly spaced (10<sup>1</sup>, 10<sup>2</sup>, 10<sup>3</sup>, 10<sup>6</sup>)

And your suggestion for the alias concatenation is great! I'll get started on this